### ISSUE TITLE
Biodiversity data rescue in the framework of a long-term Kenya-Belgium cooperation in marine sciences

### ISSUE THREAD

**Issue by kcopas** [gbif/data-mobilization#152]
New in Scientific Data:

Knockaert C, Tyberghein L, Goffin A, Vanhaecke D, Ong’anda H, Wakwabi EO & Mees J (2019) Biodiversity data rescue in the framework of a long-term Kenya-Belgium cooperation in marine sciences. Scientific Data 6: 85. https://doi.org/10.1038/s41597-019-0092-8

> The Kenya-Belgium data collection includes about 111,800 biotic observations on benthos, algae, fish, zooplankton, phytoplankton, birds and mangroves which cover more than 400 unique locations that were sampled between 1873 and 1999. The scope of this data digitization project was to recover data in theses and reports resulting from marine and coastal research activities in the Eastern African region conducted between 1984 and 1999. __Data were digitized and quality checked in the frame of the Belgian LifeWatch project__. The dataset provides a better insight into the different types of research conducted between 1985 and 1996 in frame of the Kenya-Belgium cooperation in marine sciences (KBP) project and can facilitate further coastal biodiversity research in Kenya.

Know anything about this one, @andrejjh @DimEvil @peterdesmet ? Looks like open data…

**Comment by DimEvil** [comment]
Yes, this is a Vliz initiative, check http://www.lifewatch.be/en/marine-data-archeology

**Comment by kcopas** [comment]
Very cool!

Is any of that flowing through to EurOBIS, perhaps? Not seeing anything that old coming through to GBIF, though they all seem of a piece with some of the mid-century-ish datasets from VLIZ.

**Comment by DimEvil** [comment]
There is a mention of AfroBis o.. Data can be downloaded here: http://www.vliz.be/en/imis?dasid=5821&doiid=331

It's a CC-BY so... :) I think this data should be on GBIF.... I could ask some of the authors as I know them...
